cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -682,24 +683,16 @@ def test_can_get_correct_usage_info_for_content_groups(self):
     @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
     def test_can_get_correct_usage_info_with_orphan(self, module_store_type):
         """
-        Test if content group json updated successfully with usage information even if there is
-        an orphan in content group.
+        Test if content group json updated successfully with usage information
+        even if there is an orphan in content group.
         """
         self.course = CourseFactory.create(default_store=module_store_type)
         self._add_user_partitions(count=1, scheme_id='cohort')
-        vertical, problem = self._create_problem_with_content_group(cid=0, group_id=1, name_suffix='0')
+        vertical, problem = self._create_problem_with_content_group(cid=0, group_id=1, name_suffix='0', orphan=True)
 
-        # Assert that there is no orphan in the course yet.
-        self.assertEqual(len(self.store.get_orphans(self.course.id)), 0)
-
-        # Update problem(created earlier) to an orphan.
-        with self.store.branch_setting(ModuleStoreEnum.Branch.published_only):
```

Comment:
@Qubad786 , I'd love to know here what's getting cached in the request cache only in old mongo that this method of creating an orphan fails.

Looking at it again it looks a little weird that we've only orphaned this on the published, not draft branch as well. My new method orphans it on both branches.